README.md

```diff
@@ -244,3 +271,63 @@ const wrapper = mount(
 #### Updating CRA
 
 When updating Create React App it's important to take a look at the `optimization.minimizer` values in the webpack config and then update the config in `craco.config.js`. After copying over any updates be sure to re-introduce the `terserOptions.mangle.reserved` key and values in the newly updated config.
+
+## Releasing the Dashboard
+
+To complete QA and release the Dashboard will take aproximately 2h of total time, 1h of which is an automated process at the end.
+
+- [Release prep](#release-prep)
+- [Generate a release tarball for Juju](#generate-a-release-tarball-for-juju)
+- [Release to jaas.ai](#release-to-jaas.ai)
+- [Release to Juju Simplestreams](#release-to-juju-simplestreams)
+- [Create an announcement on the Juju Discourse](#create-an-announcement-on-the-juju-discourse)
+
+### Release prep
+
+To prepare to release:
+
+- Clone a new copy of the repository to remove the chance of any uncommitted artifacts ending up in the release.
+- Thoroughly [QA the Dashboard](#qa'ing-the-dashboard) following the [Release tests](#release-tests)
+- Update the changelog by adding the important commits from the last tag.
+
+```
+  git log `git describe --tags --abbrev=0`..HEAD --format='- %b' --merges | sed '/^- $/d'
```

Comment:
Did you look into the system maas-ui uses?

Reply:
I did and it looks pretty good, there are a few others I've tried to but as far as the scope for this document I just included a simple snippet. We can certainly iterate on this process over time.